docs/installation.rst

```diff
@@ -0,0 +1,9 @@
+Installation
+------------
+
+Current ethPM CLI is only available via Pypi. Plans to make it available via Homebrew & Docker are in progress.
```

Comment:
I'd drop the "Plans to make it..." part.  I'd rather deliver the functionality than make a promise...